.github/workflows/tests.yml

```diff
@@ -8,16 +8,23 @@ on:
     types: [opened, synchronize, reopened]
 
 jobs:
-  build:
+  pytest:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
-      max-parallel: 4
+      fail-fast: false
+      # max-parallel: 6
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        os: [windows-2019, ubuntu-20.04, macOS-10.15] 
+        python-version:  [3.6, 3.7, 3.8] # 3.5 fails as pipenv installs black as a dev dep for which 3.5 is not supported
```

Comment:
Related to https://github.com/OpenMined/PyDP/pull/280/files#r481971889. If the issue is Windows and black, can't we just ignore 3.5 testing just for windows until we make a fix?

Reply:
I completely agree with this.

Reply:
I resolved this as we actually don't need black for running these tests. --skip-pipfile so we don't install it at all. Least invasive fix I could think of.